weather-map-wk7/server/server.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const cors = require('cors');
+require('dotenv').config();
+const dataWeather = require("./data")
+
+const app = express();
+
+const PORT = 8080;
+app.use(cors());
+app.use(express.json());
+
+// creates an endpoint for the route /api
+app.get('/', (req, res) => {
+    res.json({ message: 'Hello from My template ExpressJS' });
+  });
+
+// creates an endpoint for the route /api/weather
+app.get('/weather/', (req, res) => {
+  const city = req.query.cityName;
+  const apiKey= process.env.API_KEY;
+  const params = new URLSearchParams ({
+    q: city,
+    appid: apiKey,
+    units: "Imperial"
```

Comment:
America! 🇺🇸 Whoo! 💥 
Glad to see you played around with this option and updated the descriptions on the front-end to match the new output.